primitives/core/src/assertion/mod.rs

```diff
@@ -143,6 +143,15 @@ pub enum Assertion {
 	Dynamic(DynamicParams)
 }
 
+pub const A8_SUPPORTED_NETWORKS: [Web3Network; 6] = [
```

Comment:
maybe a bit picky, it's better to not use `pub` here. It should be valid only within this file.